openid4java/1.0.0.wso2v6/pom.xml

```diff
@@ -0,0 +1,192 @@
+<!--
+  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's update to correct license year (2022)

Reply:
updated